spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigServerConfigDataLoader.java

```diff
@@ -57,6 +57,11 @@
 
 public class ConfigServerConfigDataLoader implements ConfigDataLoader<ConfigServerConfigDataResource>, Ordered {
 
+	/**
+	 * PropertySource name for the config client.
+	 */
+	public static final String CONFIG_CLIENT_PROPERTYSOURCE_NAME = "configClient";
```

Comment:
Move to health indicator?

Reply:
🤦‍♂️ I meant to actually use that variable in that class, it looks like that is where we are creating the PropertySource, I just pushed the change

Reply:
👍🏻